File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
import java.util.*;
class Solution {
    public int maxFrequencyElements(int[] nums) {
        HashMap<Integer,Integer> array=new HashMap<>();
        int max=0;
        for(int i:nums)
        {
            array.put(i,array.getOrDefault(i,0)+1);
            if(array.get(i)>max)
                max=array.get(i);
        }
        int count=0;
        for(int i:array.values())
        {
            if(i==max)
                count=count+max;
        }
        return count;
    }
}

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        m={}
        max=0
        for i in nums:
            m[i]=m.get(i,0)+1
            if(m[i]>max):
                max=m[i]
        count=0
        for i in m.values():
            if(i==max):
                count=count+1
        return count*max